docs/openapi/schemas/Verification.yml

```diff
@@ -4,15 +4,16 @@ properties:
   title:
     type: string
     enum: 
-      - Proof
       - Activation
       - Expired
+      - Proof
       - Revocation
+      - Signature
   status:
     type: string
     enum: 
-      - good
-      - bad
+      - Passed
+      - Failed
```

Comment:
This hasn't been discussed? Did you pick this out of Paul's example on https://github.com/w3c-ccg/traceability-interop/issues/469? 
While arguably marginally more human intuitive than `good`/`bad` it's an annoying little "paper cut" change, and I question if it's worth the pain.

Reply:
I'm not tied to the change, reverted.